lib/rules/cli/4.1.yaml

```diff
@@ -177,9 +177,10 @@
     :from_literal: --from-literal=<value>
     :name: <value>
 :create_cronjob:
-  :cmd: oc create cronjob <name>
```

Comment:
@zhouying7780 I don't think you need to add `<global_options>` since they are global, all rules will have access to those.

Reply:
Perhaps he needs argument ordering where command follows any other options. Like with `exec`. Correct?

Reply:
@pruan-rht @akostadinov yes , just need to argument the ordering.

Reply:
Any conclusion now ?